src/style.css

```diff
@@ -1 +1,40 @@
-/* Styles go here */
+@font-face {
+  font-family: "Segoe UI";
+  src: local("Segoe UI Light");
```

Comment:
have you checked Figma design?
font-family is different there
![roboto - not segoe](https://user-images.githubusercontent.com/39308717/178443199-75ef1ed4-4864-410a-861b-a6e9fc23c710.png)

use google fonts as advised in readme: https://fonts.google.com/specimen/Roboto?query=roboto , `Medium 500`

and generally follow the readme guidelines top-to-bottom